verify_rpms/cmd_runner.py

```diff
@@ -0,0 +1,23 @@
+from subprocess import run, CompletedProcess, CalledProcessError
+from typing import List
+from verify_rpms.exceptions import CmdError
+
+
+class CmdRunner:
+
+    @staticmethod
+    def run_cmd(args: List[str]) -> CompletedProcess:
+        """
+        Run a cmd command using subprocess run method.
+        Checked flags: check, text, capture_output
+        :param args: arguments to pass to the command line
+        :return: results of the subprocess
+        :throws: CmdError
+        """
+        try:
+            result: CompletedProcess = run(args=args, check=True, text=True, capture_output=True)
```

Comment:
I think you can inject the `run` callable as a dependency and avoid some monkeypatching.

Reply:
Lets start with this implementation, we can change it later if needed.
WDYT?